crates/modelardb_server/src/main.rs

```diff
@@ -41,6 +41,15 @@ use crate::storage::StorageEngine;
 #[global_allocator]
 static ALLOC: snmalloc_rs::SnMalloc = snmalloc_rs::SnMalloc;
 
+const PORT: i16 = 9999;
+
+/// The different possible node types, assigned when the node is started.
+#[derive(Clone, PartialEq)]
```

Comment:
It seems that `Eq` requires that it also derives `PartialEq`. It also seems that `Eq` is "only informing the compiler that this is an equivalence relation rather than a partial equivalence relation" while `PartialEq` is for supporting actual equality operations. I have added `Eq` to the derive and kept `PartialEq` as well.